add-web-tiled-layer/src/main/java/com/esri/arcgismaps/sample/addwebtiledlayer/MainActivity.kt

```diff
@@ -39,18 +39,13 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         lifecycle.addObserver(mapView)
 
-        // list of sub-domains
-        val subDomains = listOf("a", "b", "c", "d")
-
         // build the web tiled layer from stamen url and subDomains
```

Comment:
This comment needs updating:
```suggestion
        // build the web tiled layer from ArcGIS Living Atlas of the World tile service url and subDomains
```